ingestors/kafka/src/main/scala/hydra/kafka/config/TopicMetadataV2ConfigSupport.scala

```diff
@@ -0,0 +1,7 @@
+package hydra.kafka.config
+
+import hydra.common.Settings.applicationConfig
+
+object TopicMetadataV2ConfigSupport {
+  val AllowedOrganizations= applicationConfig.getString("subject.allowed.organizations")
```

Comment:
Instead of going through the application config, and using the reference.conf files, you should use AppConfig.Scala and pull in configurations that way.